src/main.rs

```diff
@@ -35,7 +28,7 @@ async fn main() -> Result<(), BoxedError> {
     opts.optopt(
         "l",
         "local-port",
-        "The local port to which tcpproxy should bind to, randomly chosen otherwise",
+        "The local port to which tcp-proxy should bind to, randomly chosen otherwise",
```

Comment:
Any reason for changing this?

Reply:
> Any reason for changing this?

It was an automatic spellcheck fix, I'll change that